basics/Main.java

```diff
@@ -7,23 +7,24 @@
 
     public static void main(String[] args){
 
-        int dogCount = 0;
-        System.out.println(pluralize("dog",dogCount));
-
+        int dogCount = 1;
+        System.out.println("I own " + dogCount + " " + pluralize("dog", dogCount) + ".");
+        int catCount = 2;
+        System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
         flipNHeads(3);
         clock();
     }
 
     public static String pluralize(String word, int num){
 
         if(num==1){
-            return "I own " + num + " " + word + ".";
+            return  word;
         }
         else if(num<0){
```

Comment:
It would have been ideal to allow users to input 0 indicating they don't own any of that thing.